### ISSUE TITLE
Mobile App- UI Design

### ISSUE THREAD

**Issue by shillinc-osu** [OPEnSLab-OSU/Nanolux#54]
## Background

The AudioLux mobile app will be primarily used by event technicians and deaf/hard of hearing individuals. Therefore, accessibility should be a primary feature of the app, and consulting potential users is essential to developing a good UI.

### Functional

- The mobile app must contain the same features as the web app.
- The UI must be optimized for mobile devices.
- The mobile app UI must have feature parity with the web app UI.

### Nonfunctional

- The mobile app UI should be designed in Figma.
- The mobile app UI must be accessible to both non-technical users and the hard of hearing.
- The mobile app UI should have the same screen contents as the web app.

## Subtasks

- [ ] Design the paths for configuring the network
- [ ] Design the paths for selecting and configuring LED strip patterns
- [ ] Demo to project partner for final approval.

## Dependencies

- This story is not blocked by or blocking other user stories.

## Estimate

The Figma design should be complete no later than the end of Fall term, including presentations to the Project Partner.

## Acceptance Criteria

- [ ] The mobile app UI has screens and paths that can "configure" the network settings of the device, including AudioLux password and target network.
- [ ] The mobile app can "select patterns" and configure settings associated with the pattern.
- [ ] The mobile app has feature parity with the web app.
- [ ] The UI demo is approved by the project partner.
